themes/nekonight/nekonight.theme.sh

```diff
@@ -1,55 +1,26 @@
 # nekonight Bash prompt with source control management
 # Author: Bruno Ciccarino <brunociccarinoo@gmail.com>
 #
-#
 # Theme inspired by:
 #  - Bash_it cupcake theme
 # Demo:
 # ╭─🐱 virtualenv 🐱user at 🐱host in 🐱directory on (🐱branch {1} ↑1 ↓1 +1 •1 ⌀1 ✗)
 # ╰λ cd ~/path/to/your-directory
 
+if [ -z "${NEKONIGHT_BASE_LOADED}" ]; then
+  source "$OSH/themes/nekonight/nekonight-base.sh"
+  export NEKONIGHT_BASE_LOADED=true
+fi
```

Comment:
```suggestion
source "$OSH/themes/nekonight/nekonight-base.sh"
```

We haven't be doing the check for reloading. If we'd want to do it, I'd rather prepare a utility function for the include-once and use it in all similar themes.